src/utils/use-transportation-color.ts

```diff
@@ -46,3 +47,27 @@ export const useThemeColorForTransportMode = (
       return 'transport_other';
   }
 };
+
+export function useTransportationLineColors(fareProductType?: string) {
+  const {theme} = useTheme();
+  const {fareProductTypeConfigs} = useFirestoreConfiguration();
+
+  const fareProductTypeConfig = fareProductTypeConfigs.find(
+    (c) => c.type === fareProductType,
+  );
+
+  const {mode, subMode} = fareProductTypeConfig?.transportModes?.[0] || {};
+
+  let lineColor =
+    mode === 'water'
+      ? theme.static.background.background_2.background
+      : theme.static.background.background_accent_0.background;
```

Comment:
I think this logic on `lineColor` doesn't really make sense for us to have. I asked Christian about it.

Reply:
so that means that should be always `theme.static.background.background_accent_0.background` o should we remove it completely?

Reply:
He said `background_2.background` can always be used.